examples/eks/v1beta1/testhooks/check-clusterauth.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -aeuo pipefail
+
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+echo "obtain kubeconfig from ClusterAuth connection secret"
+${KUBECTL} -n upbound-system get secret sample-eks-cluster-conn -o go-template='{{ .data.kubeconfig | base64decode }}' > sampleclusterkube
```

Comment:
nit: We may consider [reading the namespace from the environment](https://github.com/crossplane-contrib/provider-upjet-aws/blob/4a2b3710bd93ad7ebfd2ee9375fc65d4551bccc7/Makefile#L191) but let's do this in a separate PR in which we implement this for other scripts as well (if there are any).

Reply:
the namespace is actually coming from the `writeConnectionSecretToRef` section of the particular example we are using, it is not necessarily the `CROSSPLANE_NAMESPACE`. That is why I choose to hardcode it. Though, we can consider parametrizing the namespace name at the example's `writeConnectionSecretToRef` if you prefer.